app.ts

```diff
@@ -25,28 +25,17 @@ app.use("/static", express.static(path.join(__dirname, "static")));
 app.use(express.json());
 app.use(express.urlencoded({ extended : true}))
 
-app.use(expressSession({
-    secret : "NOT HARDCODED SECRET",
-    resave : true,
-    saveUninitialized : true,
-    cookie : {maxAge : 60000},
-
-}))
+app.use(expressSession({secret : "NOT HARDCODED SECRET", cookie : {maxAge : 60000}}))
 
 declare module "express-session" {
     interface SessionData {
+        token: string;
     }
 }
+app.use('/', router);
 
-let port : number;
-if (!process.env.UI_PORT || isNaN(parseInt(process.env.UI_PORT))) {
-    port = 3000;
-}
-else {
-    port = parseInt(process.env.UI_PORT);
-}
-app.listen(port, () => {
-    console.log(`Server listening on port ${port}`);
```

Comment:
In the cloud the app has to run on a different port so you may not want to delete this

Reply:
Have changed back to env var